Webshop/webContent/web/src/Admin/Product/EditProduct.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+    <div id="editGame">
+        <h3>Product aanpassen</h3>
+        <div v-show="platformsLoaded" class="row">
+            <div class="row">
+                <div class="six columns">
+                    <label for="GameTitle"><b>Titel</b></label><input type="text" v-model="GameTitle" id="GameTitle" v-bind:value="game.GameTitle">
+                </div>
+            </div>
+            <div class="row">
+                <div class="six columns">
+                    <label for="GameRatingPegi"><b>PEGI</b></label><input type="text" v-model="GameRatingPegi" id="GameRatingPegi" v-bind:value="game.RatingPEGI">
+                </div>
+            </div>
+            <div class="row">
+                <div class="twelve columns">
+                    <div v-for="(image, index) in game.Image">
+                        <label for="GameImages" v-if="index == 0"><b>Afbeeldingen</b><input id="GameImages" type="text" v-model="GameImageValue" v-bind:value="game.Image[index]"></label>
+                    </div>
+                </div>
+            </div>
+            <div class="row">
+                <div class="twelve columns">
+                    <label for="GameDescription"><b>Omschrijving</b></label>
+                    <textarea class="u-full-width" name="GameDescription" v-model="GameDescription" id="GameDescription" cols="100" v-bind:value="game.Description"></textarea>
+                </div>
+            </div>
+            <div class="row">
+                <div class="six columns">
+                    <label for="GameMinPlayers"><b>Minimum spelers</b></label><input type="number" v-model="GameMinPlayers" id="GameMinPlayers" v-bind:value="game.MinPlayers">
+                </div>
+                <div class="six columns">
+                    <label for="GameMaxPlayers"><b>Maximum spelers</b></label><input type="number" v-model="GameMaxPlayers" id="GameMaxPlayers" v-bind:value="game.MaxPlayers">
+                </div>
+            </div>
+            <div class="row">
+                <div class="six columns">
+                    <label for="GameEAN"><b>EAN</b></label><input type="number" v-model="GameEAN" id="GameEAN" v-bind:value="game.GameEAN">
```

Comment:
I'm using an ID for every input, because a label requires a for, which points to the id of the element